cli/assets/templates/assemblyscript/asconfig.json

```diff
@@ -5,15 +5,18 @@
   "options": {
     "binaryFile": "build/cart.wasm",
     "runtime": "incremental",
+    "sourceMap": true,
     "importMemory": true,
     "initialMemory": 1,
     "maximumMemory": 1,
     "noExportMemory": true,
+    "zeroFilledMemory": true,
```

Comment:
Thanks for implementing this new feature in AS! It's really great for WASM-4 and other projects that import memory :smile: